pkg/templates/cloudwatch-agent-config.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
So this file should drive all the metrics being uploaded to CW via the agent, so any custom metrics or metrics captured via `StatsD` or `collectd` would also be defined here. 

Looking for ideas on how to make this dynamic/configurable by the user:
  * have users provide their own via another CLI arg
  * interactive mode within IQ that'll basically invoke the CW Agent Configuration Wizard
  * leave this file as-is (collecting Basic metrics) and introduce a different mechanism for subscribing to other metrics

Reply:
A basic setup like this seems good to me.  If a user is bringing their own AMI, they may very well already have the Cloudwatch Agent installed and configured as well, for that case the user should just disable cloudwatch agent installation and rely on their own. Wdyt?

Reply:
Yeah, I think that's a good user experience, but the main concern here is communicating to IQ which metrics to benchmark (i.e. use as target-utilization) which is one reason why I wanted the metrics to be known/valid when IQ starts up.

With that said, it's also possible to create another arg/config in JSON that encapsulates `metrics` and it could be a string of custom-named metrics that the user knows should be emitted by CWA, but the downside being more opportunity for failure (fat finger a name, forgot to configure it, etc.).

Who knew finding a balance between flexibility, UX, and functionality would be difficult

Reply:
discussed offline:
- by offering this flexibility we put onus on users to provide valid/existing metrics
- pass in `metrics` as another JSON object in config
- users can either provide metrics they want XOR use the default metrics in this file

Reply:
these changes will be addressed in separate PR